src/lib/integration_test_cloud_engine/coverage_manager.ml

```diff
@@ -0,0 +1,55 @@
+(* Submodule responsible for managing test coverage files. Responsible for code coverage raw files generatation from pods.
+   Requires tailored mina images with instrumentation and BISECT_SIGTEM env variable set.
+   Function traverse all applicable pods (those which has mina process) and kills them.
+   This operation dumps raw bissect files (.coverage) and downloads it to gcloud bucket.
+   WARNING: operations of test coverage retrieval is destructive (as it kills mina processes).
+*)
+open Integration_test_lib
+open Core
+
+module Coverage_manager = struct
+  type t = { logger : Logger.t }
+
+  let create ~logger = { logger }
```

Comment:
Do we really need a manager to run a single function, which is always immediately detached? It would be better to delete this and use the function as standalone.